perftracker/templates/comparison_id.html

```diff
@@ -364,7 +465,74 @@ <h4>Job: {{ job }}</h4>
     $(window).on('DOMContentLoaded load resize scroll', delayed_render);
     $('.collapse').on("shown.bs.collapse hidden.bs.collapse", delayed_render);
     delayed_render();
+
+    var keyUpStack = [];
+
+    $(".panel__input-search_text").keyup(function(e) {
+        keyUpStack.push(1);
+        $('h4').unmark();
+
+        setTimeout(function() {
+            keyUpStack.pop();
+
+            if(keyUpStack.length === 0) {
+                var buf = this.value;
+
+                var filteredList = {};
```

Comment:
misleading name, it is 'hiddenList' actually.
Alsy why you use dicts, not arrays?
Also you don't need that at all, it is sufficient to use 1 set or array for 'matched'